### DISCUSSION TITLE
Improve field editor to support configuring filter editor

### DISCUSSION THREAD

**Discussion by B-Esmaili** [directus/directus#14483] [Feature Requests]
Provide users with an extra tab in field creation wizard to select and interface type for filters , just as we do right now in interface tab.Something like following:

![Screenshot_20220718_100630](https://user-images.githubusercontent.com/9152515/179451436-24785e04-3818-4628-9d6c-ac397370c1cf.png)

**Comment by rijkvanzanten** [comment]
That feels like _a lot_ of additional configuration for a relatively small enhancement 🤔 I'm wondering if we can achieve the same benefits by smartly reusing the interface options when appropriate

**Reply by B-Esmaili** [reply]
Yes i was thinking about same thing. We can take the configured interface for filter as well by default, but if we want to have different value for all of distinct configuration items then we kinda have to replicate same configurations for filter interface.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
